lib/case.js

```diff
@@ -64,6 +71,7 @@ module.exports = (metatests) => {
           result = target;
         }
 
+        let sResult;
         if (result instanceof RegExp) {
           sResult = result.toString() + '';
```

Comment:
`result.toString() + ''` looks strange

Reply:
I didn't change this. Though might as well refactor.